tests/integration/test_array.py

```diff
@@ -0,0 +1,190 @@
+# Copyright 2022 NVIDIA Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import numpy as np
+import pytest
+from legate.core import LEGATE_MAX_DIM
+
+import cunumeric as num
+
+SCALARS = (
+    0,
+    -10.5,
+    1 + 1j,
+)
+
+ARRAYS = (
+    [],
+    (1, 2),
+    ((1, 2),),
+    [(1, 2), (3, 4.1)],
+    (
+        [
+            1,
+            2.1,
+        ],
```

Comment:
will `black` not keep this? It's weird that it would reformat this list but not the following list
```suggestion
        [1, 2.1],
```